release-notes/8.0/supported-os.json

```diff
@@ -0,0 +1,345 @@
+                    ]
+                },
+                {
+                    "id": "tvos",
+                    "name": "tvOS",
+                    "link": "https://developer.apple.com/tvos/",
+                    "lifecycle": "https://support.apple.com/apple-tv/",
+                    "architectures": [
+                        "Arm64",
+                        "x64"
+                    ],
+                    "supported-versions": [
+                        "17",
+                        "16",
+                        "15",
+                        "14",
+                        "13",
+                        "12.2"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "Linux",
+            "distributions": [
+                {
+                    "id": "alpine",
+                    "name": "Alpine",
+                    "link": "https://alpinelinux.org/",
+                    "lifecycle": "https://alpinelinux.org/releases/",
+                    "architectures": [
+                        "Arm32",
+                        "Arm64",
+                        "x64"
+                    ],
+                    "supported-versions": [
+                        "3.20",
+                        "3.19",
+                        "3.18",
+                        "3.17"
+                    ],
+                    "unsupported-versions": [
+                        "3.16"
+                    ]
+                },
+                {
+                    "id": "debian",
+                    "name": "Debian",
+                    "link": "https://www.debian.org/",
+                    "lifecycle": "https://wiki.debian.org/DebianReleases",
+                    "architectures": [
+                        "Arm32",
+                        "Arm64",
+                        "x64"
+                    ],
+                    "supported-versions": [
+                        "12"
+                    ]
+                },
+                {
+                    "id": "fedora",
+                    "name": "Fedora",
+                    "link": "https://fedoraproject.org/",
+                    "lifecycle": "https://fedoraproject.org/wiki/End_of_life",
+                    "architectures": [
+                        "Arm32",
+                        "Arm64",
+                        "x64"
+                    ],
+                    "supported-versions": [
+                        "40",
+                        "39"
+                    ],
+                    "unsupported-versions": [
+                        "38"
+                    ]
+                },
+                {
+                    "id": "opensuse",
+                    "name": "openSUSE Leap",
+                    "link": "https://www.opensuse.org/",
+                    "lifecycle": "https://en.opensuse.org/Lifetime",
+                    "architectures": [
+                        "Arm64",
```

Comment:
According to https://github.com/dotnet/core/blob/main/release-notes/8.0/supported-os.md openSUSE Leap is only supported on `x64`.

Reply:
This is more of a historicalism. We should support all the architectures where we have an offering. There is no intent to only support x64 for any of the distros.